package.json

```diff
@@ -26,13 +26,12 @@
   },
   "homepage": "https://github.com/soos-io/soos-sast#readme",
   "dependencies": {
-    "@soos-io/api-client": "^0.2.7",
+    "@soos-io/api-client": "^0.2.11",
```

Comment:
update to 0.2.12 once published 
https://github.com/soos-io/soos-api-client/pull/18/files